x-pack/libbeat/management/simple_input_config_test.go

```diff
@@ -133,7 +133,7 @@ func TestSimpleInputConfig(t *testing.T) {
 		fmt.Sprintf(":%d", server.Port),
 		"",
 		client.VersionInfo{},
-		grpc.WithTransportCredentials(insecure.NewCredentials()))
+		client.WithGRPCDialOptions(grpc.WithTransportCredentials(insecure.NewCredentials())))
```

Comment:
Can the test in here be modified to verify that chunking works as expected?

Reply:
Doesn't make sense to test a feature of an external library. The library handles all the chunking and ensures that its on, and that library is fully tested.

Reply:
In this case where there are essentially no changes besides the dependency this seems reasonable.

I am a bit worried that nothing is actually testing that this works with a real Beat anywhere though. If we can force an existing integration test in the agent repository to use chunking for everything that's probably a better way to test this.